src/components/Card/Toolbar.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box, PropsWithBox } from "@saleor/macaw-ui-next";
+import React from "react";
+
+export const Toolbar: React.FC<PropsWithBox<{ children: React.ReactNode }>> = ({
+  children,
+  ...rest
+}) => <Box {...rest}>{children}</Box>;
+
+export const toolbarWrapperStyles = {
+  display: "flex",
+  flexDirection: "row",
+  alignItems: "center",
+  justifyContent: "space-between",
+} as const;
```

Comment:
question: Can you explain me why? :D You should pass this props to Box inside Toolbar component

Reply:
Alternatively, you could add props to DashboardCard.Header for example `withToolbar` and then apply those styles or those styles should be used there by default